app/enterprise/0.32-x/kong-implementation-checklist.md

```diff
@@ -61,19 +61,19 @@ In this section, you will find a summary about the recommended network and firew
 
 These are the port settings in order for Kong to work: 
 
-- Allow HTTP traffic to [proxy_listen](/latest/configuration/#proxy_listen). By default  8000. 
-- Allow HTTPS traffic to [proxy_listen_ssl](/latest/configuration/#proxy_listen_ssl). By default 8443.
-- Allow HTTP traffic to [admin_listen](/latest/configuration/#admin_listen). By default 8001.
-- Allow HTTPS traffic to [admin_listen_ssl](/latest/configuration/#admin_listen_ssl). By default 8444.
+- Allow HTTP traffic to [proxy_listen](/0.13.x/configuration/#proxy_listen). By default  8000. 
+- Allow HTTPS traffic to [proxy_listen_ssl](/0.13.x/configuration/#proxy_listen_ssl). By default 8443.
```

Comment:
This jumps to the top of the page. There's no anchor link.

Reply:
There may be a separate issue with anchor links OSS of this PR. Hard link works correctly: https://docs.konghq.com/0.13.x/configuration/#proxy_listen

Reply:
Ah, you were referring to /#proxy_listen_ssl. It seems this property is entirely absent. Will check with Core.

Reply:
```suggestion
```